app/logic/Abstract/Message.ts

```diff
@@ -85,6 +90,24 @@ export class Message extends Observable {
     }
     return null;
   }
+
+  get htmlWithExternal(): string {
```

Comment:
This doesn't work, because `EMail.html` overwrites `Message.html` and also downloads the msg.

Also, we don't copy complex functions like this.

Rather, I'd suggest to make a flag `withExternalContent = false` on the msg, and then check in it the `html` getter.